mvc/src/main/java/com/interface21/webmvc/servlet/mvc/tobe/HandlerExecution.java

```diff
@@ -1,12 +1,23 @@
 package com.interface21.webmvc.servlet.mvc.tobe;
 
+import com.interface21.webmvc.servlet.ModelAndView;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import com.interface21.webmvc.servlet.ModelAndView;
+import java.lang.reflect.Method;
 
 public class HandlerExecution {
 
+    private final Method method;
+
+    public HandlerExecution(Method method) {
+        this.method = method;
+    }
+
     public ModelAndView handle(final HttpServletRequest request, final HttpServletResponse response) throws Exception {
-        return null;
+        return (ModelAndView) method.invoke(
+                method.getDeclaringClass().getDeclaredConstructor().newInstance(),
+                request,
+                response
+        );
```

Comment:
```suggestion
        return (ModelAndView) method.invoke(
                obj,
                request,
                response
        );
```

`handle(final HttpServletRequest request, final HttpServletResponse response)` 메서드를 처리할 때 매번 새로운 인스턴스를 생성하는 것 같아요! 이를 효율적으로 바꿔볼 수 있을까요?

Reply:
오 고려하지 못했었는데 좋은 부분 말씀해주셔서 감사합니다 👍 
HandlerExecution를 생성할 때 특정 컨트롤러와 그 메서드를 가질 수 있도록 반영해 해결 했습니다 !